pkgs/by-name/po/pocket-casts/package.nix

```diff
@@ -44,11 +44,12 @@ buildNpmPackage rec {
   ];
 
   postInstall = ''
-    install -Dm644 $out/lib/node_modules/pocket-casts/icon.png $out/share/pixmaps/pocket-casts.png
-    install -Dm644 $out/lib/node_modules/pocket-casts/icon-x360.png $out/share/pixmaps/pocket-casts-x360.png
+    install -Dm644 $out/lib/node_modules/pocket-casts/img/icon-x512.png $out/share/pixmaps/pocket-casts-x512.png
+    install -Dm644 $out/lib/node_modules/pocket-casts/img/icon-x360.png $out/share/pixmaps/pocket-casts-x360.png
 
     makeWrapper ${electron}/bin/electron $out/bin/pocket-casts \
-      --add-flags $out/lib/node_modules/pocket-casts/main.js
+      --add-flags $out/lib/node_modules/pocket-casts/main.js \
+      --set ELECTRON_FORGE_DISABLE_CONSOLE_LOGGING true
```

Comment:
Why `ELECTRON_FORGE_DISABLE_CONSOLE_LOGGING` out of interest?

Reply:
`ELECTRON_FORGE_DISABLE_CONSOLE_LOGGING` no longer in the diff, marking this one as resolved.